### ISSUE TITLE
Unsupported Block Type

### ISSUE THREAD

**Issue by bramus** [terraform-google-modules/terraform-google-service-accounts#12]
When following the basic example one is greeted with this error:

```
$ terraform plan

Error: Unsupported block type

  on .terraform/modules/service_account--cloudfunctions/terraform-google-modules-terraform-google-service-accounts-6256d48/outputs.tf line 41, in data "template_file" "keys":
  41:   vars {

Blocks of type "vars" are not expected here. Did you mean to define argument
"vars"? If so, use the equals sign to assign it a value.
```

This is due to the example referring to an older version, incompatible with the latest terraform version

```
version = "~> 0.1"
```

When using the latest version (`2.0.0` atm) of `terraform-google-modules/terraform-google-service-accounts`, everything works fine.

**Comment by ludoo** [comment]
Can you share Terraform version and how you are calling the module?

**Comment by aaron-lane** [comment]
Fixed by #13.
